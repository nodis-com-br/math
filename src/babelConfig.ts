export const moduleResolver = [
  'module-resolver',
  {
    root: ['.'],
    alias: {
      '@': './src/',
    },
  },
];
