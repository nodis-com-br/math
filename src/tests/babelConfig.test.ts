import { moduleResolver } from '../babelConfig';

describe('babelConfig', () => {
  test('should match snapshot', () => {
    expect(moduleResolver).toMatchSnapshot();
  });
});
