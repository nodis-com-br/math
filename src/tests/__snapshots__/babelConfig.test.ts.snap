// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babelConfig should match snapshot 1`] = `
Array [
  "module-resolver",
  Object {
    "alias": Object {
      "@": "./src/",
    },
    "root": Array [
      ".",
    ],
  },
]
`;
